website/docs/d/policy_vms.html.markdown

```diff
@@ -15,6 +15,7 @@ This data source is applicable to NSX Policy Manager and VMC.
 
 ```hcl
 data "nsxt_policy_vms" "all" {
+  state      = "running"
```

Comment:
do we need to also add guest_os to the example template here?

Reply:
Added that as well